PokemonAPI/PokemonAPI/Controllers/TrainerController.cs

```diff
@@ -31,7 +32,15 @@ public async Task<IActionResult> GetAll(CancellationToken cancellationToken = de
         [HttpGet("{id}")]
         public async Task<IActionResult> Get(Guid id, CancellationToken cancellationToken = default)
         {
-            return Ok(await _trainerService.Get(id, cancellationToken));
+            try
+            {
+                return Ok(await _trainerService.Get(id, cancellationToken));
+            }
+            catch
+            {
+                throw new NotFoundException(id);
+            }
+            
```

Comment:
Revert these changes. This is just leftover from testing. Unauthorized exception is already thrown at service level. No need to do try catch blocks in our controllers